website/src/components/Homepage/Features/FeatureList.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
```

Comment:
I suggest you implement the description with markdown instead of React. 
Then you can remove the react. It will be a static configuration rather React components